app/experimenter/experiments/constants/nimbus.py

```diff
@@ -47,6 +47,7 @@ class Version(models.TextChoices):
         FIREFOX_99 = "99.0"
         FIREFOX_100 = "100.0"
 
+    MAX_SLUG_LEN = 80
```

Comment:
Can you add a comment that says what you said in the ticket, i.e. 
`Telemetry systems including Firefox Desktop Telemetry v4 and Glean have limits on the length of their unique identifiers,
we should limit the size of our slugs to the smallest limit, which is 80 for Firefox Desktop Telemetry v4.`

Reply:
Why not put the `MAX_SLUG_LEN` under the comment?

Reply:
Is there a preferred ordering between constants and comments I don't know about?

Reply:
In general I'm used to comments describing things immediately under/after them, this is the first time I've seen it in reverse. 🤷‍♂️